command-line-interface.md

```diff
@@ -0,0 +1,189 @@
+# Operations
+
+A conformant [runtime][] MUST provide an executable (called `funC` in the following examples).
```

Comment:
Let's change this to something like:
```
A conformant runtime MAY expose a command line interface/executable (called `funC` in the following examples).
```
Not all conformant runtime need to support a cli - they could be embedded within a bigger system but still be compliant to the runtime spec.